index.js

```diff
@@ -991,7 +992,25 @@ module.exports = function (log, indexesPath) {
     } else if (!op.type) {
       // to support `query(fromDB(jitdb), toCallback(cb))`
       getFullBitset(cb)
-    } else console.error('Unknown type', op)
+    } else if (op.type === 'DEFERRED') {
+      // DEFERRED only appears in this pipeline when using `prepare()` API
+      op.task(
+        { jitdb },
+        (err, newOp) => {
+          if (err) {
+            console.error('Error executing DEFERRED operation', err)
+            cb(null, new TypedFastBitSet())
+            return
+          }
+          if (newOp) {
+            getBitsetForOperation(newOp, cb)
+          } else {
+            cb(new TypedFastBitSet())
```

Comment:
should be: `null, new TypedFastBitSet()`, right?

Reply:
Same as above :)
cb is `cb(bitset, filters)`